src/templates/blog-template.js

```diff
@@ -97,10 +97,7 @@ BlogPostLink.propTypes = {
 }
 
 const Blog = ({ pageContext }) => {
-  const { currentPage, isFirstPage, isLastPage, totalPages, contentfulData } = pageContext
-  const nextPage = `/blog/${String(currentPage + 1)}`
-  const prevPage = currentPage - 1 === 1 ? '/blog' : `/blog/${String(currentPage - 1)}`
-
+  pageContext.path = 'blog'
```

Comment:
Please update `path` to be a prop on the Pagination component remove `pageContext.path`

Reply:
done